be/src/common/config.cpp

```diff
@@ -1081,6 +1081,9 @@ DEFINE_mInt64(cache_lock_held_long_tail_threshold_us, "30000000");
 DEFINE_mBool(enable_file_cache_keep_base_compaction_output, "false");
 DEFINE_mInt64(file_cache_remove_block_qps_limit, "1000");
 DEFINE_mInt64(file_cache_background_gc_interval_ms, "100");
+DEFINE_mInt64(file_cache_background_lru_dump_interval_ms, "60000");
+DEFINE_mInt64(file_cache_background_lru_dump_tail_record_num, "5000");
+DEFINE_mInt64(file_cache_background_lru_log_replay_interval_ms, "1000");
```

Comment:
add a switch to enable/disable LRU persis, or make sure tail_record_num = 0 will do the work